players/installing-java/linux.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Installing Java on Linux
+description: A step by step guide on how to install Java on Linux.
+---
+
+# Installing Java on Linux
+
+This guide will walk you through installing Java 17 on Linux.
+
+The Minecraft Launcher comes with its own java installation, so this section is only relevant if you want to use the Fabric `.jar` based installer, or if you want to use the Minecraft Server `.jar`.
```

Comment:
On linux you will need to install java, so this section can be removed. But keep it on Windows.